src/ui/components/watchlist/CardFilter.vue

```diff
@@ -11,11 +11,8 @@
 
     <v-dialog v-model="confirmUpdate" max-width="500px" persistent>
       <v-card :loading="loading">
-        <v-card-title class="text-h5">
-          <span
-            >¿Are you sure you want to change all the filters in the
-            watchlist?</span
-          >
+        <v-card-title class="text-h5" style="word-break: break-word">
+          ¿Are you sure you want to change all the filters in the watchlist?
```

Comment:
```suggestion
          Are you sure you want to change all the filters in the watchlist?
```

'¿' no se usa en ingles